outbox/README.md

```diff
@@ -3,49 +3,75 @@
-The receiving service ("shipment-service") receives these events (and would apply some processing based on them),
+The receiving service ("shipment-service-quarkus") receives these events (and would apply some processing based on them),
 excluding any duplicate messages by comparing incoming event ids with already successfully consumed ids.
 
-**Update, March 3, 2019:** Another variant of the receiving service has been added, "shipment-service-quarkus",
-which is functionally the same as the original one, but is implemented using the [Quarkus](https://quarkus.io) stack.
-This allows to build a native binary of that service, resulting in significantly less memory usage and faster start-up than the classic version (based on Thorntail).
+Both services are implemented using the [Quarkus](https://quarkus.io) stack.
+This allows building a native binary of each service, resulting in significantly less memory usage and faster start-up than the JVM-based version.
 
-## Execution
+## Building
 
-Prepare the Java components:
+Prepare the Java components by first performing a maven build.
 
 ```console
 $ mvn clean install -Pnative -Dnative-image.docker-build=true
 ```
 
-Start all components:
+This illustates the usage of Quarkus' `native` profile mode where the quarkus-maven-plugin will generate not only JVM-based artifacts but also native images.
+The `native` profile and `native-image.docker-build` environment variable can be omitted if native image artifacts are not wanted.
+
+## Environment
+
+Setup the necessary environment variables
 
 ```console
 $ export DEBEZIUM_VERSION=0.10
+$ export QUARKUS_BUILD=native
+```
+
+The `DEBEZIUM_VERSION` specifies which version of Debezium artifacts should be used.
+The `QUARKUS_BUILD` specifies whether docker-compose will build containers using Quarkus in JVM or Native modes.
+The default is `jvm` for JVM mode but `native` can also be specified to build Quarkus native containers.
+  
+## Start the demo  
+
+Start all components:
+
+```console
 $ docker-compose up --build
 ```
 
-Register the Debezium Postgres connector:
+This executes all configurations set forth by the `docker-compose.yaml` file.
+
+It's important to note that sometimes the order or shipment service may fail to start if the dependent database takes longer than expected to initialize.  If that happens, simply re-execute the command again and it will start the remaining services. 
+
+## Configure the Debezium connector
+
+Register the connector that to stream outbox changes from the order service: 
 
 ```console
 $ http PUT http://localhost:8083/connectors/outbox-connector/config < register-postgres.json
 HTTP/1.1 201 Created
 ```
 
+## Call the various REST-based APIs
+
 Place a "create order" request with the order service:
 
 ```console
-$ http POST http://localhost:8080/order-service/rest/orders < resources/data/create-order-request.json
+$ http POST http://localhost:8080/rest/orders < resources/data/create-order-request.json
```

Comment:
While at it, let's get rid of the "/rest" part, too.

Reply:
Removed